pegasus/cache/i18n/en-US.yml

```diff
@@ -659,9 +659,8 @@
   dashboard_faq_k5: "K-5 Frequently Asked Questions"
   dashboard_announcements: "Announcements"
   dashboard_announce_view_all: "View all"
-  dashboard_announce_message: "<b>New feature</b> - Now you can move your students between your sections or to another teacher's section."
-  dashboard_announce_message_second: "<b>Phase one applications closing</b> - Do you (or a teacher you know) want to teach high school computer science? Professional learning workshops are open this summer for a full year intro course that students love. And it's designed so you can teach without any prior experience. Join our free workshops by signing up before April 22nd."
-  dashboard_announce_button_text: "Learn more"
+  dashboard_announce_message: "<b>April 22, 2016</b> - Unplugged activities are a vital part of the learning process, but teachers tell us that finding all of the resources can be confusing and time-consuming. We listened! Now, you can find all of the unplugged resources on a single landing page. Associate the lessons with any computer science class, or use stand alone as an independent course."
```

Comment:
i feel like people expect courses listed on studio to be "our" but if not, then maybe good to be consistent.